### PR TITLE
tests: unit tests for utils/filters.go and name subtests

### LINKED ISSUE #70: cli: create tests for existing functionalities and commands
After #3 is done, we should test all the functionalities and commands we already have. The following list should summarize the different components we need to test:

- [x] Utils functions
- [x] Validation functions
- [x] `root` functions and general way to test command output
- [x] `close` command
- [x] `du` command
- [x] `info` command
- [ ] `list` command
- [x] `logs` command
- [x] `open` command
- [ ] `status` command
- [x] `ls` command

### PR DESCRIPTION
**Pull request by BrunoRosendo**
closes #70

### CODE REVIEW

File: utils/filters_test.go [hunk 1]
```diff
@@ -0,0 +1,561 @@
...
+				}
+			}
+		})
+	}
+
+	filters, _ := NewFilters([]string{"single"}, []string{}, []string{})
+	err := filters.AddFilter("invalid_filter")
+	if err == nil {
+		t.Errorf("Expected error for AddFilter(%s), got nil", "invalid_filter")
+	}
```
Comment (mdonadoni) [inline utils/filters_test.go:203]: What do you think about using `t.Run(...)` also for this test?

File: utils/filters_test.go [hunk 2]
```diff
@@ -0,0 +1,561 @@
...
+					if filter != "value" {
+						t.Errorf("Expected filter value to be 'value', got '%s'", filter)
+					}
+				}
+			}
+		})
+	}
+}
+
+func TestFilters_AddFilters(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (BrunoRosendo) [reply utils/filters_test.go:99]: the IDE suggested this for some reason, but I also prefer without the `_`

File: utils/filters_test.go (hunk 1 above)
Comment (BrunoRosendo) [reply utils/filters_test.go:203]: Nice catch

**Review by mdonadoni — APPROVED** [review]
Looks good!

**Pull request merged**